.github/pull_request_template.md

```diff
@@ -0,0 +1,21 @@
+### Description
+What does this PR do?
+
+### TODO
+- [ ] Make sure there's a changelog. Brand new features have a differently
+  formatted changelog than other changelogs, so pay attention to that. If this
+  PR is the CE portion of an ENT PR, put the changelog here, _not_ in your ENT
+  PR.
+- [ ] Add a milestone. If this change is going to be backported, each backport
+  PR should receive the milestone for the exact version it's being backported
+  to. For this PR, just pick the biggest version.
+- [ ] If this PR is the CE portion of an ENT change, and that ENT change is
```

Comment:
It might be worthwhile to try to remind people to add:

1. A reference to the associated Jira, if there is one: (i.e.`Resolves: VAULT-12358132135`)
2. A link to the associated RFC (if there is one).
3. A link to the associated (ideally, approved) Enterprise PR.

Reply:
Oh and maybe some guidance about Squash Merge wouldn't be too bad.

Reply:
I added your first 3 bullet points. It was brought up that CE PRs can only be squash merged, so the last point isn't necessary.